api/index.md

```diff
@@ -14,6 +14,6 @@ This API reference includes the Q# standard library and the three libraries buil
 
 - [Q# libraries](xref:microsoft.quantum.qsharplibintro)
 - [.NET libraries](xref:microsoft.quantum.dotnetlibsintro)
-- [Python libraries](https://docs.microsoft.com/python/qsharp)
+- [Python libraries](https://docs.microsoft.com/python/qsharp-core/qsharp)
```

Comment:
This link gives me a 404 not found?

Reply:
I think that's a dependency on https://github.com/MicrosoftDocs/python-quantum/pull/22; if I understand correctly, this should not 404 once that PR is in.

Reply:
Yes, once https://github.com/MicrosoftDocs/python-quantum/pull/21 is merged to `live`, it will work.

Here's where it will go:
https://review.docs.microsoft.com/en-us/python/qsharp-core/qsharp?branch=master

Reply:
Thanks, that link is helpful! Does review.docs.microsoft.com always reflect `master` or is there a manual build step to deploy to there?

Reply:
The review site is internal-only, but can be pointed to any branch by using the dropdown box at the upper-right, or by using the `branch` parameter in the URL.

Reply:
It's an automatic process (I think takes ~2-3 minutes to re-render the content after each commit). You can preview any branch by just replacing `branch=master` at the end with the branch name that you want to preview.

There are also links in PRs to preview the rendered content, e.g. in this PR you can see those at: https://github.com/MicrosoftDocs/quantum-docs-pr/pull/1028#issuecomment-680308060

Reply:
(I really wish GitHub would live-update PR comments on this page so that I don't keep repeating the same thing as @cgranade is saying... 😂)

Reply:
Sorry about that....

Reply:
@guenp @cgranade would one of you mind approving this (if everything else looks good)? Then I'll open a new PR in this repo to merge this one to `live`, which can be merged simultaneously with https://github.com/MicrosoftDocs/python-quantum/pull/22.

Reply:
Done, Ryan! And thanks both!

Reply:
Thank you both! Now finally one last step, if you could approve https://github.com/MicrosoftDocs/quantum-docs-pr/pull/1029 we'll get both changes to live. 😊